src/NuGet.Clients/NuGet.VisualStudio.Common/NuGet.VisualStudio.Common.csproj

```diff
@@ -17,6 +17,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Build" ExcludeAssets="runtime" />
     <PackageReference Include="Microsoft.Build.Tasks.Core" ExcludeAssets="runtime" PrivateAssets="All" />
+    <PackageReference Include="System.Security.Cryptography.Xml" ExcludeAssets="runtime" PrivateAssets="All" />
```

Comment:
Sort alphabetically, and I think the MSBuild packages have `ExcludeAssets` for a reason (something would go wrong otherwise, we want MSBuild to be loaded from its default path). I guess since tests pass that using `ExcludeAssets` doesn't cause a problem, but otherwise I don't understand why ExcludeAssets is being used. The packages that are not Microsoft.Build.* below don't have Exclude assets.